packages/airnode-operation/src/evm/utils.ts

```diff
@@ -39,5 +39,5 @@ export const deriveWalletPathFromSponsorAddress = (sponsorAddress: string): stri
     const shiftedSponsorAddressBN = sponsorAddressBN.shr(31 * i);
     paths.push(shiftedSponsorAddressBN.mask(31).toString());
   }
-  return `0/${paths.join('/')}`;
```

Comment:
again, we should at least have a `protocolId` as arg here

Reply:
Do we need it anytime soon? I'd rather postpone this.

Reply:
Let's postpone it since Burak already said so https://github.com/api3dao/airnode/pull/772#pullrequestreview-854572547